packages/scoutgame/src/partnerRewards/getPartnerRewardsForScout.ts

```diff
@@ -157,7 +154,7 @@ export async function getPartnerRewards({
 
   partnerRewardPayouts.forEach((payout) => {
     const partnerReward: PartnerRewardBase<any> = {
-      points: Number(formatUnits(BigInt(payout.amount), payout.payoutContract.tokenDecimals)),
+      tokens: Number(formatUnits(BigInt(payout.amount), payout.payoutContract.tokenDecimals)),
```

Comment:
are we handling it properly when you have a decimal payout? (ie maybe calling toFixed() in the frontend)?